src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -93,8 +95,17 @@ private void InitializeColors()
             CodeAnalysisColors.AccentBarColorKey = EnvironmentColors.FileTabInactiveDocumentBorderEdgeBrushKey;
         }
 
-        protected override void LoadComponentsInUIContext(CancellationToken cancellationToken)
+        protected override async Task LoadComponentsInUIContextAsync(CancellationToken cancellationToken)
         {
+            await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
+
+            await GetServiceAsync(typeof(SVsTaskStatusCenterService)).ConfigureAwait(true);
+            await GetServiceAsync(typeof(SVsErrorList)).ConfigureAwait(true);
+            await GetServiceAsync(typeof(SVsSolution)).ConfigureAwait(true);
+            await GetServiceAsync(typeof(SVsShell)).ConfigureAwait(true);
```

Comment:
How do these help us? These are already going to have been fetched by somebody prior to Roslyn; if we're ignoring the result, what's the point?

Reply:
:bulb: We should file a follow-up issue to improve the way we initialize these. The example I gave in the other commit is one possible direction, but likely others are available as well.

Reply:
filed https://github.com/dotnet/roslyn/issues/29598